iltasiivo.php

```diff
@@ -698,11 +698,18 @@ function is_log($str) {
               AND varaston_hyllypaikat.hyllyvali      = tuotepaikat.hyllyvali
               AND varaston_hyllypaikat.hyllytaso      = tuotepaikat.hyllytaso
               AND varaston_hyllypaikat.reservipaikka  = 'K')
+            LEFT JOIN inventointilistarivi ON (inventointilistarivi.yhtio = tuotepaikat.yhtio
```

Comment:
Mitä jos kuitenkin lisäis inventointilistarivi.tuotepaikkatunnus-kentän, niin sais nää queryt näteiks, mut sit tarpeen mukaan jos joskus tilataan sellanen softa jossa luodaan inventointilistakopio jostain vanhasta listasta, niin voisi lukea hyllytiedot suoraan inventointilistarivi-taulusta?